cucumber-rails.gemspec

```diff
@@ -47,7 +46,10 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.4.0'
   s.rubygems_version = '>= 1.6.1'
-  s.files            = Rake::FileList['**/*'].exclude(*File.read('.gitignore').split)
-  s.test_files       = Dir['{spec,features}/**/*']
   s.require_path     = 'lib'
+  s.files            = Dir.chdir(File.expand_path(__dir__)) do
+                         `git ls-files -z`.split("\x0").select do |file|
+                           file.match(%r{^(lib/|CHANGELOG|CONTRIBUTING|LICENSE|README)})
+                         end
+                       end
```

Comment:
Thanks, however, this change was done explicitly to not use `git` in gemspec, which is problematic for downstream maintainers and even in general; cf: https://docs.rubocop.org/rubocop-packaging/cops_packaging.html#packaginggemspecgit

This is not only Debian-specific but applies to most of the downstream maintainers.

That said, I'd be happy to shift from `Rake::FileList` to `Dir` (which is [what I personally use](https://github.com/utkarsh2102/rubocop-packaging/blob/master/rubocop-packaging.gemspec#L21)). So maybe more like `Dir["lib/**/*", "CHANGELOG", ...]` should be a good replacement here.

What do you think?

Reply:
I'm happy to remove references to `git`. 👍

Reply:
I agree - we should not depend on `git` when building the gemspec.

Reply:
Seems we have a consensus :D

Reply:
Added in 89b14cf. The resulting gem file has identical contents as the `git` version.